roles/ocp/tasks/creds.yml

```diff
@@ -32,3 +32,11 @@
         dest: "{{ lookup('env', 'HOME') }}/.{{ creds_path }}/{{ creds_file }}"
         mode: 0600
   when: not cloud_creds.stat.exists
+
```

Comment:
There are a few notes that needs to be addresses.

Currently, the `ocp` role supports the following platform:
- aws
- gcp
- osp

Both aws and osp has a single file that is able to contain multiple profiles for the cloud.
While for gcp we should create a separate file with profile content and use it.

The purpose of this change is that we would like to avoid deleting the profiles that the user already has.

In that case, use a `blockinfile` module to create or update the configuration files for aws and osp.
And use a `template` module to create a separate file for gcp. Name it `gcp-service-account-rhacm-automation.json`.

The reference for the profiles happens in the following way:
For aws, use an environment variable `AWS_PROFILE` in `Process OCP clusters` task with the name of the profile.
For gcp, use an environment variable `GOOGLE_APPLICATION_CREDENTIALS` and point it to the created file.
For osp, the name of the profile defined within `install-config.yml` file of openshift. Make sure to update it there.

Set the name of the profile for aws and osp as `rhacm-automation`.
It will make sure that the profile for the automation will not delete any default profile used by the user.